jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -54,12 +54,23 @@ def functionTest(String test_name, String TEST_GROUP, Boolean RUN_FIT_TEST, Bool
             try{
                 timeout(90){
                     // run test script
-                    sh '''#!/bin/bash -x
-                    pushd build-config
-                    ./build-config
-                    popd
-                    ./build-config/test.sh
-                    '''
+                    def repos = env.REPOS_UNDER_TEST.tokenize(',')
```

Comment:
@sunnyqianzhang , can you please provide the sandbox job link which prove image-service PR Gate can work.